### PR TITLE
Add doc for Read the Docs service

### PR DESCRIPTION
**Pull request by dpoirier**
DJAN-88

### CODE REVIEW

File: source/services/read-the-docs.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+You can link the RTD account to GitHub accounts (or Bitbucket,
+etc) to let it access the source of a private project, but RTD
+can't really protect the built documentation from unwanted
+access.
+
+According to the RTD `privacy controls <http://docs.readthedocs.org/en/latest/privacy.html>`_,
+you can set a level of "private", but it just hides the project
+from lists and searches. Anyone with the URL can still view
+the documentation.
```
Comment (ironfroggy) [inline source/services/read-the-docs.rst]: Can we include any guidance on the on 1) alternative hosting or 2) deciding if this level of privacy is sufficient for a non-public project?

Comment (dpoirier) [inline source/services/read-the-docs.rst]: I added a few more words, though I don't know if they'll fully address this concern.

**Comment by dpoirier** [comment]
Any more comments?

**Comment by ironfroggy** [comment]
+1

**Pull request merged**